apps/stream-deck/net.progwise.timebook.sdPlugin/propertyInspector/tracking_pi.html

```diff
@@ -0,0 +1,31 @@
+<!doctype html>
+<html>
+  <head lang="en">
+    <meta charset="utf-8" />
+    <title>net.progwise.timebook PI</title>
+    <script src="https://cdn.jsdelivr.net/gh/geekyeggo/sdpi-components@v2/dist/sdpi-components.js"></script>
+  </head>
+  <body>
+    <sdpi-wrapper>
+      <sdpi-item label="Task">
+        <sdpi-select
+          setting="taskID"
```

Comment:
```suggestion
          setting="taskId"
```

and in TypeScript as well!